2.0/README.md

```diff
@@ -0,0 +1,476 @@
+# Vector Tile Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in
+this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## 1. Purpose
+
+This document specifies a space-efficient encoding format for tiled geographic vector data. It is designed to be used in browsers or server-side applications for fast rendering or lookups of feature data.
+
+## 2. File Format
+
+The Vector Tile format uses [Google Protocol Buffers](https://developers.google.com/protocol-buffers/) as a encoding format. Protocol Buffers are a language-neutral, platform-neutral extensible mechanism for serializing structured data.
+
+## 2.1. File Extension
+
+The filename extension for Vector Tile files SHOULD be `mvt`. For example, a file might be named `vector.mvt`.
+
+## 2.2. Multipurpose Internet Mail Extensions (MIME)
+
+When serving Vector Tiles the MIME type SHOULD be `application/vnd.mapbox-vector-tile`.
+
+## 3. Projection and Bounds
+
+A Vector Tile represents data based on a square extent within a projection. A Vector Tile SHOULD NOT contain information about its bounds and projection. The file format assumes that the decoder knows the bounds and projection of a Vector Tile before decoding it.
+
+[Web Mercator](https://en.wikipedia.org/wiki/Web_Mercator) is the projection of reference, and [the Google tile scheme](http://www.maptiler.org/google-maps-coordinates-tile-bounds-projection/) is the tile extent convention of reference. Together, they provide a 1-to-1 relationship between a specific geographical area, at a specific level of detail, and a path such as `https://example.com/17/65535/43602.mvt`.
+
+Vector Tiles MAY be used to represent data with any projection and tile extent scheme.
+
+## 4. Internal Structure
+
+This specification describes the structure of data within a Vector Tile. The reader should have an understanding of the [Vector Tile protobuf schema document](vector_tile.proto) and the structures it defines.
+
+### 4.1. Layers
+
+A Vector Tile consists of a set of named layers. A layer contains geometric features and their metadata. The layer format is designed so that the data required for a layer is contiguous in memory, and so that layers can be appended to a Vector Tile without modifying existing data.
+
+A Vector Tile SHOULD contain at least one layer. A layer SHOULD contain at least one feature.
```

Comment:
Not sure about why we would say that. *"A Vector Tile SHOULD contain at least one layer"* may be wrong just because some area is covered by no layer, and *"A layer SHOULD contain at least one feature"* may be wrong when some area is not including any feature. So, as for my understanding, those two sentences should be removed.